core/crypto/src/vrf.rs

```diff
@@ -157,182 +149,30 @@ impl SecretKey {
     }
 }
 
-macro_rules! peq {
-    (Proof, $this:expr, $other:expr) => {
-        $this.0[..] == $other.0[..]
-    };
-    ($ty:ident, $this:expr, $other:expr) => {
-        $this.0 == $other.0
-    };
-}
-
-macro_rules! as_bytes {
-    (SecretKey, $this:expr) => {
-        $this.0.as_bytes()
-    };
-    ($ty:ident, $this:expr) => {
-        &$this.0
-    };
-}
-
-macro_rules! to_str {
-    ($v:expr) => {
-        identity::<String>($v.into()).as_str()
-    };
-}
-
 macro_rules! traits {
```

Comment:
Why is this macro still in `vrf.rs`?

Reply:
Because this macro is specific to `vrf.rs`, while others are also used for randomness.